warhammer-core/WarhammerCore.Business/UserService.cs

```diff
@@ -51,5 +67,21 @@ public async Task<string> SignUpAsync(string email, string password)
 
             return userId;
         }
+
+        private string GenerateJSONWebToken(UserInfo user)
+        {
+            var securityKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(_config["Jwt:Key"]));
+            var credentials = new SigningCredentials(securityKey, SecurityAlgorithms.HmacSha256);
+            var claims = new List<Claim>();
+            claims.Add(new Claim("email", user.Email));
```

Comment:
This can be simplified:

```suggestion
             var claims = new List<Claim>
            {
                new Claim("email", user.Email)
            };
```

or 
```suggestion
            var claims = new List<Claim> { new Claim("email", user.Email) };
```